backend/benefit/applications/api/v1/serializers.py

```diff
@@ -729,6 +733,40 @@ def get_applicant_terms_in_effect(self, obj):
         else:
             return None
 
+    def get_warnings(self, obj):
+        """
+        Return a list of warnings. The data format is same as for error responses.
+        For warnings related to former benefit, the key used is "former_benefits"
+
+        More fields may be added in the future. The format of the data is:
+            {
+                "some_field_name_or_key": [
+                    "warning_string"
+                ],
+                "other_field_name_or_key": [
+                    "warning_string"
+                ],
+            }
```

Comment:
Looking at the code the returned lists as the dictionary values here may be of arbitrary non-empty length but the documentation seems to suggests that they're always of length 1.

Reply:
Hmm I'll make the documentation more clear about that